modules/rosa-delete-objects.adoc

```diff
@@ -0,0 +1,278 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-delete-objects_{context}"]
+= Delete objects
+
+[role="_abstract"]
+This section describes the `delete` commands for clusters and resources.
+
+[id="rosa-delete-admin_{context}"]
+== delete admin
+
+Deletes a cluster administrator from a specified cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete admin --cluster=<cluster_name>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Example
+Delete a cluster administrator from a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa delete admin --cluster=mycluster
+----
+
+[id="rosa-delete-cluster_{context}"]
+== delete cluster
+
+Deletes a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete cluster --cluster=<id> | <name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to delete.
+
+|--watch
+|Watches the cluster uninstallation logs.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+
+|--yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Examples
+Delete a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa delete cluster --cluster=mycluster
+----
+
+[id="rosa-delete-idp_{context}"]
+== delete idp
+
+Deletes a specific identity provider (IDP) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete idp <id> | <name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster from which the IDP will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+
+|--yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Example
+Delete an identity provider named `github-1` from a cluster named `mycluster`.
```

Comment:
Should we use something more generic than `github-1`?

Reply:
Since it's an IDP and not an IDP user, I changed it to `github`.